### PR TITLE
Fix copy-pasta in Bigquery docs.

### PR DESCRIPTION
**Pull request by tseaver**

### CODE REVIEW

File: docs/bigquery-usage.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ Authorization / Configuration
   your applications.
 
 - :class:`Client <gcloud.bigquery.client.Client>` objects hold both a ``project``
-  and an authenticated connection to the PubSub service.
+  and an authenticated connection to the Bigquery service.
```
Comment (dhermes) [inline docs/bigquery-usage.rst]: s/Bigquery/BigQuery

File: gcloud/bigquery/connection.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 
 class Connection(base_connection.JSONConnection):
-    """A connection to Google Cloud Pubsub via the JSON REST API."""
+    """A connection to Google Cloud Bigquery via the JSON REST API."""
```
Comment (dhermes) [inline gcloud/bigquery/connection.py]: s/Bigquery/BigQuery

File: docs/bigquery-usage.rst (hunk 1 above)
Comment (tseaver) [inline docs/bigquery-usage.rst]: bf33188

File: gcloud/bigquery/connection.py (hunk 2 above)
Comment (tseaver) [inline gcloud/bigquery/connection.py]: bf33188

**Comment by tseaver** [comment]
@dhermes PTAL.

**Comment by dhermes** [comment]
LGTM

**Pull request merged**